src/main/kotlin/frc/robot/subsystems/hood/HoodConstants.kt

```diff
@@ -1,80 +1,44 @@
 package frc.robot.subsystems.hood
 
-import com.ctre.phoenix6.configs.*
-import com.ctre.phoenix6.signals.GravityTypeValue
 import com.ctre.phoenix6.signals.InvertedValue
-import com.ctre.phoenix6.signals.StaticFeedforwardSignValue
 import edu.wpi.first.math.geometry.Translation2d
 import edu.wpi.first.math.geometry.Translation3d
 import edu.wpi.first.units.*
-import frc.robot.lib.webconstants.LoggedTunableNumber
+import frc.robot.Constants
+import frc.robot.lib.Gains
+import frc.robot.lib.LoggedTunableNumber
+import frc.robot.lib.createGains
 
 object HoodConstants {
-    const val GEAR_RATIO : Double = 3.0 * (36.0 / 18.0) * (158.0 / 18.0)
+    const val GEAR_RATIO: Double = 3.0 * (36.0 / 18.0) * (158.0 / 18.0)
     const val ENCODER_TICKS_PER_REVOLUTION = 4096.0
     const val SIMULATION_LENGTH = 3.0
-    const val MAX_TOLERANCE : Double = 0.75/360
+    const val MAX_TOLERANCE: Double = 0.75 / 360
```

Comment:
Use WPILib conversion functions (and maybe measures)

Reply:
Those are percentages and looking at a tolerance constant as a percent is way less readable than seeing them as an amount of degrees.

Reply:
That's not a matter of readability, it's just a different method. 
If you use `isNear`, you'll need to use percentage, though.